install/streams.csv

```diff
@@ -1077,6 +1077,8 @@ IF07,10,E,H,100,false,false,false,2008-10-07T04:31:00Z,2009-02-11T22:00:00Z
 IF10,10,E,H,100,false,false,false,2008-10-01T03:14:00Z,2009-02-11T23:00:00Z
 IF12,10,E,H,100,false,false,false,2008-10-03T01:46:00Z,2009-02-11T23:30:00Z
 IF14,10,E,H,100,false,false,false,2008-10-08T23:31:00Z,2009-02-13T01:30:00Z
+IFMZ,10,H,H,100,false,false,false,2025-03-16T22:26:00Z,9999-01-01T00:00:00Z
```

Comment:
IFMZ HH is axial right ? (as in the data queried)
Orienation to set in mind :)

Reply:
Yes 10m borehole = axial